examples/benchmark/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2016, Dennis Sitelew
+cmake_minimum_required(VERSION 3.0)
+
+if(NOT CMAKE_BUILD_TYPE)
+  message(WARNING "You shouldn't benchmark a debug build")
```

Comment:
@yowidin Interestingly enough the benchmark itself also have such warning: https://github.com/google/benchmark#debug-vs-release , https://github.com/google/benchmark/blob/1b263fe6d906bb0854b84247f1b395bbacd3b88e/src/reporter.cc#L53

Ok, maybe devs indeed should make such warning to themselves.